pkg/plugins/backendplugin/instrumentation/instrumentation.go

```diff
@@ -65,7 +65,7 @@ func instrumentPluginRequest(ctx context.Context, cfg Cfg, pluginCtx *backend.Pl
 	pluginRequestDuration.WithLabelValues(pluginCtx.PluginID, endpoint, string(cfg.Target)).Observe(float64(elapsed / time.Millisecond))
 	pluginRequestCounter.WithLabelValues(pluginCtx.PluginID, endpoint, status, string(cfg.Target)).Inc()
 
-	PluginRequestDurationSeconds.WithLabelValues("backend-datasource", pluginCtx.PluginID, endpoint, string(cfg.Target), status).Observe(float64(elapsed / time.Second))
+	PluginRequestDurationSeconds.WithLabelValues("grafana-backend", pluginCtx.PluginID, endpoint, string(cfg.Target), status).Observe(elapsed.Seconds())
```

Comment:
Nit: Swap the definition order of labels for `PluginRequestDurationSeconds` so that they follow the same as `pluginRequestDuration` IE

```diff
-[]string{"source", "plugin_id", "endpoint", "target", "status"})
+[]string{"source", "plugin_id", "endpoint", "status", "target"})
```